my-webxr-app/src/App.tsx

```diff
@@ -91,7 +90,6 @@ export default function App() {
   return (
     <>
       <div>
-        {import.meta.env.VITE_IS_TESTING && <TestingOptions />}
```

Comment:
The testing menu was accidentally removed 😱 I've got it back in the Rollbar PR

Reply:
Ahh thanks for catching that Zander! We were dealing with some merge conflicts earlier today so it must've been accidentally deleted.